README.md

```diff
@@ -52,13 +52,17 @@ If you are looking for a quickstart, we can recommend
 
 ## Rococo Deployment
 
-We currently have a live deployment of the Canvas parachain on Rococo, Polkadot's
-parachain testnet. You can access the network through Polkadot JS Apps by [clicking here](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fcanvas-rpc.parity.io#/explorer).
+We have a live deployment of the Canvas parachain on Rococo ‒ a testnet for Polkadot and Kusama parachains.
+You can access the network through Polkadot JS Apps,
+[click here for a direct link to Canvas](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fcanvas-rpc.parity.io#/explorer).
 
-The Canvas parachain uses Rococo's native token (ROC) instead of having its own token.
+The Canvas parachain uses the Rococo relaychain's native token (ROC) instead of having its own token.
 Due to this you'll need ROC in order to deploy contracts on Canvas.
 
-You can get ROC testnet tokens through the [Rococo Faucet Channel on Element](https://matrix.to/#/#rococo-faucet:matrix.org).
+As a first step, you should create an account. See [here](https://wiki.polkadot.network/docs/learn-account-generation)
+for a detailed guide. The shortcut is to just use `subkey generate`.
```

Comment:
We shouldn't be pointing users to `subkey`, especially for playing around on a testnet. If anything we point them to the Polkadot JS extension.

```suggestion
As a first step, you should create an account. See [here](https://wiki.polkadot.network/docs/learn-account-generation)
for a detailed guide.
```